backend/README.md

```diff
@@ -7,35 +7,51 @@ In order to have all the secret keys set up properly, you must create a `.env` f
 Example setup:
 
 In `.env`:
+
 ```
-MONGO_UURL=mongodb://mongouser:mykey@somedb.mlab.com:port/db
-MONGO_DURL=mongodb://mongouser:mykey@somedb.mlab.com:port/db
-MONGO_URL=mongodb://mongouser:mykey@somedb.mlab.com:port/db
+MONGO_URL=mongodb://db:27017/endpoint
```

Comment:
Nit: maybe renaming `endpoint` to something else to not confuse, as what comes after the `/` will be the [default database name](https://docs.mongodb.com/manual/reference/connection-string/) rather than a API-like request URL.

Reply:
ohh i see, it shouldn't be the actual mongo_url tho cuz that's a security issue? so i should do smth like /<default-db-name>?